### PR TITLE
Backend auth integration

### PR DESCRIPTION
**Pull request by rglasgow-deloitte**
Add supporting pages
- 401 Unauthorized Response
- 403 Forbidden Response

Update the api endpoints with credentials passed from the backend service. Removes the use of user tokens and authorization headers for api calls.

### CODE REVIEW

File: __tests__/pages/403.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const renderer = () => {
+
+  return render(
+    <MemoryRouterProvider>
+        <Forbidden />
+    </MemoryRouterProvider>
+  );
+};
+
+describe('401 Page', () => {
```
Comment (rglasgow-deloitte) [inline __tests__/pages/403.test.js:15]: This title doesn't match the name of the file

**Comment by rglasgow-deloitte** [comment]
We still have failing tests. These need to be resolved prior to merging.

File: __tests__/pages/401.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    jest.useFakeTimers();
+    jest.spyOn(global, 'setTimeout');
+    const { getByText } = renderer();
+    new Promise((r) => setTimeout(r, 2000));
+    act(() => {
+        const button = getByText(/Click Here to be Redirected/i);
+        fireEvent.click(button);
+      });
+    
+    expect(getByText('Click Here to be Redirected')).toBeInTheDocument();
```
Comment (pabastien) [inline __tests__/pages/401.test.js:36]: Behavior after button is clicked does not seem to be getting checked.

**Review by pabastien — COMMENTED** [review]
Looks good to me, just a few clarifications needed. (see comments)

File: pages/403.js [hunk 3]
```diff
@@ -1,26 +1,30 @@
 import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { useState, useEffect } from 'react';
 
 export default function Forbidden({ errorMessage }) {
-    
     // should hang on page for 15 seconds
     // should show count down to redierect 
     // should have a button to go home
     // redirect at end of countdown to home.
```
Comment (rglasgow-deloitte) [inline pages/403.js:9]: Lets go ahead and remove this otherwise looks good!

**Review by rglasgow-deloitte — COMMENTED** [review]
We still have some failing tests. I'll look into them 
- search page
- course page
- searchResult component
- shareButton component

**Review by pabastien — COMMENTED** [review]
Looks good!

**Review by pabastien — APPROVED** [review]

**Pull request merged**